tests/contrib/validation/jsonschema/test_tools.py

```diff
@@ -0,0 +1,41 @@
+import os
+import sys
+
+import pytest
+from spidermon.contrib.validation.jsonschema.tools import get_schema_from
+
+
+@pytest.mark.skipif(
+    sys.version_info < (3, 5), reason="we don't want to test json on different versions"
```

Comment:
JSON seem to return different errors for Python 2 and 3, so considering we will abandon it support soon I prudently decided to skip it.